_rules/SC1-3-1-aria-required-owned-element.md

```diff
@@ -0,0 +1,150 @@
+---
+name: ARIA required owned elements
+
+description: |
+	The rule checks existence of role(s) on required owned elements.
+
+success_criterion:
+- 1.3.1
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Audrey Maniez
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) with a [non empty](#non-empty) `role` attribute whose value is applicable to required owned elements.
+
+**Note:** I
+
+### Expectation
+
+Each target element has atleast one owned element with a relevant `role` attribute.
+
+## Assumptions
+
+*There are currently no assumptions/ add any assumptions as necessary*
+
+## Accessibility Support
+
+*There are no major accessibility support issues known for this rule/ or add any support details as necessary*
+
+## Background
+
+- [Required Owned Element](https://www.w3.org/TR/wai-aria-1.1/#mustContain)
+
+## Test Cases
+
+### Passed
+
+#### Pass example 1
+
+Owned `ul` element with role `list` has `span` element with role `listitem`.
+
+```html
+<ul role='list'>
+	<span role='listitem'>
+	</span>
+	<span>
+		<!-- atleast one element is expected have a role, not mandatory on all children -->
+	</span>
+</ul>
+```
+
+#### Pass example 2
+
+Owned `ol` element with role `tablist` has `li` element with role `tab`.
+
+```html
+<ol role='tablist'>
+	<li role='tab'>
+	</li>
+</ol>
+```
+
+#### Pass example 3
+
+Owned `ul` element with role `list` has `li` element which has an implcit semantic role.
```

Comment:
I do not think this will fail @annethyme. I tested that an `li` item gets an implicitRole of `listitem` irrespective of what is it nested under & the case you have defined, where the div has an explicit role of `list` means, this should pass.